### DISCUSSION TITLE
change working directory for projects

### DISCUSSION THREAD

**Discussion by wypifu** [mfussenegger/nvim-dap#1041] [General]
Hi, I am used to gdb and I wanted to move to nvim-adp-ui has it looks nicer. However, with gdb i can change the working directory before 'run' usually I worki like this:

file $(path/to/executable)/a.out
cd $(path/to/working/dir)
run
In nvim I start vim-adp-ui and it asked me for excutable path then start running. has I didn't set the working directory my program exist with error as it can't find the files required. I've been looking for a wait to set cwd looks like I need to do it in the config file I am not sure about it but if it is the case I am more lost as I need to change the working directory per program.

In short is it possible to change the working directory per project/executions without touching the config files like I do with gdb?
